lib/croods/resource/json_schema/definitions.rb

```diff
@@ -4,8 +4,6 @@ module Croods
   module Resource
     module JsonSchema
       module Definitions
-        TYPES = { datetime: 'string', text: 'string', json: 'object' }.freeze
```

Comment:
Isn't this used anymore?

Reply:
https://seasonedworkspace.slack.com/archives/CFGF6Q551/p1588368177012800